demos/rag_agentic/notebooks/Level0_getting_started_with_Llama_Stack.ipynb

```diff
@@ -34,76 +34,80 @@
+      "Requirement already satisfied: oauthlib>=3.2.2 in /opt/app-root/lib64/python3.11/site-packages (from kubernetes->llama-stack==0.2.4) (3.2.2)\n",
+      "Requirement already satisfied: numpy>=1.23.2 in /opt/app-root/lib64/python3.11/site-packages (from pandas->llama-stack-client==0.2.4) (2.1.3)\n",
+      "Requirement already satisfied: pytz>=2020.1 in /opt/app-root/lib64/python3.11/site-packages (from pandas->llama-stack-client==0.2.4) (2025.1)\n",
+      "Requirement already satisfied: tzdata>=2022.7 in /opt/app-root/lib64/python3.11/site-packages (from pandas->llama-stack-client==0.2.4) (2025.1)\n",
+      "Requirement already satisfied: wcwidth in /opt/app-root/lib64/python3.11/site-packages (from prompt-toolkit->llama-stack-client==0.2.4) (0.2.13)\n",
+      "Requirement already satisfied: charset-normalizer<4,>=2 in /opt/app-root/lib64/python3.11/site-packages (from requests->llama-stack==0.2.4) (3.4.1)\n",
+      "Requirement already satisfied: markdown-it-py>=2.2.0 in /opt/app-root/lib64/python3.11/site-packages (from rich->llama-stack-client==0.2.4) (3.0.0)\n",
+      "Requirement already satisfied: pygments<3.0.0,>=2.13.0 in /opt/app-root/lib64/python3.11/site-packages (from rich->llama-stack-client==0.2.4) (2.19.1)\n",
+      "Requirement already satisfied: regex>=2022.1.18 in /opt/app-root/lib64/python3.11/site-packages (from tiktoken->llama-stack==0.2.4) (2024.11.6)\n",
+      "Requirement already satisfied: cachetools<6.0,>=2.0.0 in /opt/app-root/lib64/python3.11/site-packages (from google-auth>=1.0.1->kubernetes->llama-stack==0.2.4) (5.5.1)\n",
+      "Requirement already satisfied: pyasn1-modules>=0.2.1 in /opt/app-root/lib64/python3.11/site-packages (from google-auth>=1.0.1->kubernetes->llama-stack==0.2.4) (0.4.1)\n",
+      "Requirement already satisfied: rsa<5,>=3.1.4 in /opt/app-root/lib64/python3.11/site-packages (from google-auth>=1.0.1->kubernetes->llama-stack==0.2.4) (4.9)\n",
+      "Requirement already satisfied: mdurl~=0.1 in /opt/app-root/lib64/python3.11/site-packages (from markdown-it-py>=2.2.0->rich->llama-stack-client==0.2.4) (0.1.2)\n",
+      "Requirement already satisfied: pyasn1<0.7.0,>=0.4.6 in /opt/app-root/lib64/python3.11/site-packages (from pyasn1-modules>=0.2.1->google-auth>=1.0.1->kubernetes->llama-stack==0.2.4) (0.6.1)\n",
+      "\n",
+      "\u001b[1m[\u001b[0m\u001b[34;49mnotice\u001b[0m\u001b[1;39;49m]\u001b[0m\u001b[39;49m A new release of pip is available: \u001b[0m\u001b[31;49m24.2\u001b[0m\u001b[39;49m -> \u001b[0m\u001b[32;49m25.1\u001b[0m\n",
+      "\u001b[1m[\u001b[0m\u001b[34;49mnotice\u001b[0m\u001b[1;39;49m]\u001b[0m\u001b[39;49m To update, run: \u001b[0m\u001b[32;49mpip install --upgrade pip\u001b[0m\n"
      ]
     }
    ],
    "source": [
-    "!pip install llama-stack-client==0.2.2 llama-stack==0.2.2"
+    "!pip install llama-stack-client==0.2.4 llama-stack==0.2.4"
```

Comment:
Can we remove this? Any python requirements should be defined in the `pyproject.toml`

Reply:
based on our discussion, I think we agreed to have this in the notebook for now?